include/cantera/thermo/IdealGasPhase.h

```diff
@@ -386,6 +386,22 @@ class IdealGasPhase: public ThermoPhase
         setDensity(p * meanMolecularWeight() / RT());
     }
 
+    //! Set the internally stored temperature of the phase (K).
+    //! Since the gas is in equilibrium, the electron temperature
+    //! is also set to the same value.
+    //! @param temp  Temperature in Kelvin
+    virtual void setTemperature(const double temp) {
```

Comment:
> `Phase::electronTemperature` is a virtual function that just returns `m_temp`

As a friendly amendment, this could just return the gas temperature. Iirc that would produce the correct thermodynamic behavior. That way, a reaction that uses `TwoTempPlasmaRate` can be used without restrictions to specific thermo phases.

Reply:
That is exactly what I suggested.

Reply:
I’m not sure that `m_temp` is even needed outside of `Plasma`

Reply:
`m_temp` is the variable in class `Phase` that has always been used to store the (normal) temperature.

Reply:
Not sure what got into me here!!